Makefile

```diff
@@ -677,19 +672,13 @@ mock-grpc-server-image: mock-grpc-server-build clean-image
 $(CURDIR)/image/postgres/bundle.tar.gz:
 	/usr/bin/env DEBUG_BUILD="$(DEBUG_BUILD)" $(CURDIR)/image/postgres/create-bundle.sh $(CURDIR)/image/postgres $(CURDIR)/image/postgres
 
-.PHONY: $(CURDIR)/image/postgres/Dockerfile.gen
-$(CURDIR)/image/postgres/Dockerfile.gen:
-	ROX_IMAGE_FLAVOR=$(ROX_IMAGE_FLAVOR) \
-	envsubst '$${ROX_IMAGE_FLAVOR}' \
-	< $(CURDIR)/image/postgres/Dockerfile.envsubst > $(CURDIR)/image/postgres/Dockerfile.gen
-
 .PHONY: central-db-image
-central-db-image: $(CURDIR)/image/postgres/bundle.tar.gz $(CURDIR)/image/postgres/Dockerfile.gen
+central-db-image: $(CURDIR)/image/postgres/bundle.tar.gz
 	docker build \
 		-t stackrox/central-db:$(TAG) \
 		-t $(DEFAULT_IMAGE_REGISTRY)/central-db:$(TAG) \
 		--build-arg ROX_IMAGE_FLAVOR=$(ROX_IMAGE_FLAVOR) \
```

Comment:
Actually, not sure why ROX_IMAGE_FLAVOR needs to be set for postgres

Reply:
Doesn't seem like it's needed there.